media-sound/mpd/patches/mpd-0.21.19.patchset

```diff
@@ -1,15 +1,20 @@
-From b1fcd53d895256d41cf95945328c57877cb3c8c6 Mon Sep 17 00:00:00 2001
-From: =?UTF-8?q?Fran=C3=A7ois=20Revol?= <revol@free.fr>
-Date: Thu, 31 Jul 2014 23:09:29 +0200
-Subject: Haiku: add icon as resource definition file
+From 98b34984d4d8e463d006a3c052f519639c508260 Mon Sep 17 00:00:00 2001
+From: trungnt2910 <trungnt282910@gmail.com>
```

Comment:
There are some minor changes in the patchset as far as I can tell, could you keep the credits from the original creator of these?

Reply:
Actually, the original `patchset` works fine.
I'm just merging the two changes to the .rdef file
So should I keep the old one?

Reply:
Keep the old one, merge your changes up on the old patchset, that way the both of you will have/keep their credits for the changes :)

Reply:
I don't know if it can work, give me a while to test packaging with the old and renamed patchset.

Reply:
I've checked it, and the package builds properly. Thanks.

Reply:
Thanks!